src/CattyTests/Setup/CatrobatSetupTests.swift

```diff
@@ -57,11 +57,11 @@ final class CatrobatSetupTests: XCTestCase {
         UserDefaults.standard.set(true, forKey: kUseArduinoBricks)
 
         let categoriesArduinoEnabled = CatrobatSetup.registeredBrickCategories()
-        XCTAssertTrue(categoriesArduinoEnabled.count > categoriesPhiroEnabled.count)
+        XCTAssertTrue(categoriesArduinoEnabled.count == categoriesPhiroEnabled.count)
```

Comment:
This test does not make a lot of sense anymore. You can leave it, but please also add two additional tests verifying that the according categories are enabled/disabled according to the `UserDefaults`.

PS: Please prefer `XCTAssertEqual` over `XCTAssertTrue` since that will provide more information in case of a regression.